sections/07_conclusions.md

```diff
@@ -52,8 +52,26 @@ addition, the stochasticity and complexity of biological systems implies that
 for some problems, for instance, predicting gene regulation in disease, perfect
 accuracy will be unattainable.
 
+We are witnessing deep learning models achieving human-level performance across
```

Comment:
Just to get rid of duplicate "we": maybe "a number of biomedical domains, yet this does not mean biologists or clinicians will be out of a job anytime soon."